modules/network-observability-kafka-option.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-kafka-option_{context}"]
+
+= Installing Kafka (optional)
+The Kafka Operator is supported for large scale environments. You can install the Kafka Operator as link::(https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2)[Red Hat AMQ Streams] from the Operator Hub, just as the Loki Operator and Network Observability Operator were installed. 
```

Comment:
```suggestion
The Kafka Operator is supported for large scale environments. You can install the Kafka Operator as https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2[Red Hat AMQ Streams] from the Operator Hub, just as the Loki Operator and Network Observability Operator were installed. 
```
 it shows extra 'link::(' in preview, not sure if this suggestion is correct, but tried to follow https://docs.asciidoctor.org/asciidoc/latest/macros/url-macro/

Reply:
Oh, yeah actually I don't need the parentheses here. Or two colons.